src/components/DonationTrackingTable/DonationTrackingTable.jsx

```diff
@@ -0,0 +1,132 @@
+import { useBackend } from '../../contexts/BackendContext';
+import { useState, useEffect } from 'react';
+import DonationSite from './DonationSite';
+import { Tabs, TabList, Tab, Button } from '@chakra-ui/react';
+import { ArrowDownIcon } from '@chakra-ui/icons';
+
+import {
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  TableContainer,
+  Checkbox,
+} from '@chakra-ui/react';
+
+const DonationTrackingTable = () => {
+  const { backend } = useBackend();
+
+  const [donationTrackingTableData, setDonationTrackingTableData] = useState([]);
+  const [checkedSet, setCheckedSet] = useState(new Set());
+  const [topCheckBox, setTopCheckBox] = useState(false);
+  var filter = 'month';
+  const headers = [
+    'Donation Site',
+    'Donation ID',
+    'Food Bank',
+    'Person Reporting',
+    'Email',
+    'Date',
+    'Canned Dog Food',
+    'Dry Dog Food',
+    'Canned Cat Food',
+    'Dry Cat Food',
+    'Misc Items',
+    'Volunteer Hours',
+  ];
+
+  const tabHeaders = ['month', 'quarter', 'year', 'all'];
+
+  const getDonationTrackingTableData = async filter => {
+    const response = await backend.get(`/donation/filter/${filter}`);
+    const data = response.data;
+
+    for (let i = 0; i < data.length; ++i) {
+      data[i] = { donation_site: 'NULL', ...data[i] };
+    }
+
+    setDonationTrackingTableData(data);
+  };
+  useEffect(() => {
+    getDonationTrackingTableData(filter);
+  }, []);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has missing dependencies: 'filter' and 'getDonationTrackingTableData'. Either include them or remove the dependency array.